source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizerTests/Action.ExecuteTest.mm

```diff
@@ -82,24 +82,23 @@ - (void)testRefreshInterface
     executeAction->SetVerb("doStuff");
     const std::string dataString = "{\"HereIs\" :  \"Some Data\"}";
     executeAction->SetDataJson(dataString);
-    ACORefresh *refresh = [[ACORefresh alloc] init:[ACOBaseActionElement getACOActionElementFromAdaptiveElement:executeAction] userIds:@[@"user1", @"user2", @"user3"]];
+    ACORefresh *refresh = [[ACORefresh alloc] init:[ACOBaseActionElement getACOActionElementFromAdaptiveElement:executeAction] userIds:@[ @"user1", @"user2", @"user3" ]];
```

Comment:
>  [](http://example.com/codeflow?start=166&length=1)

extra space?

Reply:
Is it a list? if yes, it's controlled by SpacesInContainerLiterals and was expected.